control/kutrace_lib.cc

```diff
@@ -739,9 +739,9 @@ void kutrace::DoEvent(u64 eventnum, u64 arg) {::DoEvent(eventnum, arg);}
 void kutrace::DoFlush() {::DoFlush();}
 void kutrace::DoInit(const char* process_name) {::DoInit(process_name);}
 void kutrace::DoMark(u64 n, u64 arg) {::DoMark(n, arg);}
-bool kutrace::DoTest() {::DoTest();}
-bool kutrace::DoOff() {::DoOff();}
-bool kutrace::DoOn() {::DoOn();}
+bool kutrace::DoTest() { return ::DoTest(); }
+bool kutrace::DoOff() { return ::DoOff(); }
+bool kutrace::DoOn() { return ::DoOn(); }
```

Comment:
💄 The coding style in this file is to not leave a space before/after the curly braces, so `{return ::DoTest();}` and such is more appropriate, but your call.